service/create/service.go

```diff
@@ -1066,31 +1066,31 @@ func (s *Service) deleteFunc(obj interface{}) {
 			s.logger.Log("error", errgo.Details(err))
```

Comment:
We can just remove this to get rid of the errgo dependency completely by replacing `errgo.Details(err)` with `fmt.Sprintf("%#v", err)`.